microformats.go

```diff
@@ -118,13 +118,19 @@ func Parse(r io.Reader, baseURL *url.URL) *Data {
 // ParseNode parses the microformats found in doc.  baseURL is the URL this
 // document was retrieved from and is used to resolve any relative URLs.
```

Comment:
I think it's worth updating the godoc to clarify the behavior when baseURL is nil.  Something like `If baseURL is nil, relative URLs are not expanded.`  If you don't mind, maybe also change "resolve" to "expanded" in that last sentence as well, since we're not actually doing any URL resolution?

Reply:
In fact, as long as an absolute base URL is in the document (i.e. `<base href="https://example.com">` we do expand the relative URLs even if `baseURL` is `nil`.

I tried to document this (also in `Parse()` for consistency), but the wording remains clumsy IMO.